pkg/server/scrub.go

```diff
@@ -0,0 +1,35 @@
+package server
+
+import (
+	"fmt"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+// scrubbing utilities to remove user information from arbitrary objects
+
+const ScrubbedReplacement = "<scrubbed>"
```

Comment:
Does this need to be exported?

Reply:
No; I thought I might need it but ended up not
I'll make it lowercase

Reply:
Oh oh I see, my bad